packages/namekit-react/src/components/DisplayedPrice.tsx

```diff
@@ -152,15 +65,13 @@ export const DisplayedPrice = ({
           <>
```

Comment:
I don't understand why there's any `Tooltip` included in `DisplayedPrice`. I couldn't find any story where this is visible. In my mind there's no need for this. Am I missing something?

Reply:
I assumed we handling displaying the price in a Tooltip would be a good feature, let me know if you find it best to remove this variant from DisplayedPrice internal logics. 

[Here](http://localhost:6006/?path=/story/namekit-displayedprice--eth-price-in-a-tooltip) is one example of **any** trigger being able to display the formatted price in a tooltip

Reply:
@FrancoAguzzi I can't see any reason why we would want this. To me this looks like completely the opposite from the goal of composability that I keep asking you to put more attention to.

If someone wants to put the string for a price into a tooltip message, why in the world would they want to do it this way? I'm struggling to put into words just how wrong this whole direction is. Please change it!